apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -462,13 +462,24 @@ def _get_tables_via_sparksql(self, database, table_names='*'):
       self.close(handle)
 
       # We get back: database | tableName | isTemporary
-      return [{
-          'name': row[1],
-          'type': 'VIEW' if row[2] else 'TABLE',
-          'comment': ''
-        }
-        for row in result.rows()
-      ]
+      try:
+        return [{
+            'name': row[1],
+            'type': 'VIEW' if row[2] else 'TABLE',
+            'comment': ''
+          }
+          for row in result.rows()
+        ]
+      except:
+        tables = result.data_table.results.results.columns[0].stringVal.values
+        # We get back: database | tableName | isTemporary
```

Comment:
Nit: We can remove this duplicate comment? It is also present on line 464

Reply:
Will remove this.